modules/core/shared/src/main/scala/lucuma/core/util/NewType.scala

```diff
@@ -28,20 +30,21 @@ trait NewType[Wrapped]:
   val value: Iso[Type, Wrapped] = Iso[Type, Wrapped](_.value)(apply)
 
   extension (t: Type)
-    inline def value: Wrapped = t
+    inline def value: Wrapped                           = t
     inline def modifyValue(f: Wrapped => Wrapped): Type = apply(f(value))
 
-  given (using CanEqual[Wrapped, Wrapped]): CanEqual[Type, Type] = CanEqual.derived
-  given (using eq: Eq[Wrapped]): Eq[Type]                        = eq
-  given (using enc: Encoder[Wrapped]): Encoder[Type]             = enc
-  given (using dec: Decoder[Wrapped]): Decoder[Type]             = dec
-  given (using disp: Display[Wrapped]): Display[Type]            = disp
-  given (using m: Monoid[Wrapped]): Monoid[Type]                 = m
-  given (using ord: Order[Wrapped]): Order[Type]                 = ord
-  given (using ord: Ordering[Wrapped]): Ordering[Type]           = ord
+  given (using CanEqual[Wrapped, Wrapped]): CanEqual[Type, Type]               = CanEqual.derived
+  // Only provide an Eq instance if it doesn't also provide Order
+  given (using eq:   Eq[Wrapped], noOrder: NotGiven[Order[Wrapped]]): Eq[Type] = eq
```

Comment:
This one line is the actual change. The rest is just formatting.

Reply:
This works, but the recommended way to fix this is by explicit prioritization.

https://www.scala-lang.org/2024/08/19/given-priority-change-3.7.html#explicit-prioritization-by-owner

Reply:
Hmm. I see. It seems like a lot more work since we're just forwarding the instances. I can change if it seems best, though.

Reply:
I just tried and it's not nice, because the `opaque type Type` doesn't work across inheritance boundaries ... this is fine 🙈